hypothesis-python/tests/cover/test_annotations.py

```diff
@@ -78,7 +72,7 @@ def f(*, a, b=2):
         pass
 
     out = convert_positional_arguments(f, (), {"a": 1})
-    assert out == ((), {"a": 1, "b": 2})
+    assert out == ((), {"a": 1})
```

Comment:
This is a deliberate change for consistency: the previous behaviour of `convert_positional_arguments()` applied default arguments only for keyword-only params, which could lead to some weird reprs for strategies.  Fortunately it can't affect `@given()`, because there it's an explicit error to have default values!